swap/src/cli.rs

```diff
@@ -77,19 +85,32 @@ pub enum Resume {
 
         #[structopt(flatten)]
         config: Config,
+
+        #[structopt(flatten)]
+        storage: Storage,
     },
 }
 
 #[derive(structopt::StructOpt, Debug)]
 pub struct Config {
     #[structopt(
         long = "config",
-        help = "Provide a custom path to a configuration file. The configuration file must be a toml file.",
+        help = "Provide a custom path to the configuration file. The configuration file must be a toml file.",
         parse(from_os_str)
     )]
     pub config_path: Option<PathBuf>,
 }
 
+#[derive(structopt::StructOpt, Debug)]
+pub struct Storage {
+    #[structopt(
+        long = "data-dir",
+        help = "Provide a custom path to the data directory.",
+        parse(from_os_str)
+    )]
+    pub data_dir_path: Option<PathBuf>,
+}
```

Comment:
```suggestion
#[derive(structopt::StructOpt, Debug)]
pub struct DataDir {
    #[structopt(
        long = "data-dir",
        help = "Provide a custom path to the data directory.",
        parse(from_os_str)
    )]
    pub data_dir_path: Option<PathBuf>,
}
```
Be mindful of introducing new terminology when it is not necessary.